cli/enable/enable_test.go

```diff
@@ -0,0 +1,67 @@
+package enable
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/otiai10/copy"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/tarantool/tt/cli/config"
+	"github.com/tarantool/tt/cli/util"
+)
+
+func TestEnableBase(t *testing.T) {
+	tempDir, err := os.MkdirTemp("", "enable_test")
+	defer os.RemoveAll(tempDir)
+	assert.Nil(t, err)
+	err = copy.Copy("./test_data", tempDir)
+	assert.Nil(t, err)
+	cliOpts := &config.CliOpts{
+		Env: &config.TtEnvOpts{InstancesEnabled: filepath.Join(tempDir, "in_en")}}
+
+	// Test enable application.
+	appPath := filepath.Join(tempDir, "test_app")
+	err = Enable(appPath, cliOpts)
+	assert.Nil(t, err)
+	appLink, err := util.ResolveSymlink(filepath.Join(cliOpts.Env.InstancesEnabled, "test_app"))
+	assert.Nil(t, err)
+	assert.Contains(t, appLink, appPath)
+
+	// Test enable script.
+	appPath = filepath.Join(tempDir, "test_script", "test.lua")
+	err = Enable(appPath, cliOpts)
+	assert.Nil(t, err)
+	appLink, err = util.ResolveSymlink(filepath.Join(cliOpts.Env.InstancesEnabled, "test.lua"))
+	assert.Nil(t, err)
+	assert.Contains(t, appLink, appPath)
+}
+
+func TestEnableNoFile(t *testing.T) {
+	tempDir, err := os.MkdirTemp("", "enable_test")
```

Comment:
testing.T supports creating temporary directories, which will will be automatically removed.